cpp/include/graphar/util/filesystem.h

```diff
@@ -154,4 +154,3 @@ Result<std::shared_ptr<FileSystem>> FileSystemFromUriOrPath(
     const std::string& uri, std::string* out_path = nullptr);
 
 }  // namespace graphar
-
```

Comment:
why delete the empty line of the file?

Reply:
I left a blank line in my editor, and I felt a little weird coming across this situation

Reply:
Maybe your editor has configured to trim the empty line automatically? You can check about this